src/getValue.js

```diff
@@ -16,18 +16,14 @@ const getValue = (
   event: SyntheticInputEvent<*>,
   currentValue: any,
   valueProp: any,
-  isReactNative: boolean
 ) => {
   if (
-    !isReactNative &&
     event.nativeEvent &&
```

Comment:
when `event.nativeEvent.text` is present in non-RN environment (I'm assuming browser target)?